Formula/alpine.rb

```diff
@@ -1,11 +1,9 @@
 class Alpine < Formula
   desc "News and email agent"
   homepage "https://alpineapp.email"
-  url "https://alpineapp.email/alpine/release/src/alpine-2.25.tar.xz"
-  mirror "https://alpineapp.email/alpine/release/src/Old/alpine-2.25.tar.xz"
```

Comment:
I don't think we want to remove this mirror. It doesn't work now, but it'll work when the next version is released and the `url` breaks.

Reply:
Ah, I see.  Sure, I'll put it back and add a comment explaining to ignore the corresponding `brew audit --strict --online` error...